src/components/sidebar/components/sidebar-nav/style.module.css

```diff
@@ -1,6 +1,9 @@
 .sidebarNavElement {
   padding-top: 8px;
   position: relative;
+  /* necessary as otherwise the skipToMainContent
+     link causes x-overflow */
+  overflow: hidden;
```

Comment:
👀  seems the focus outline is getting clipped:
<img width="208" alt="Screen Shot 2022-01-12 at 5 22 20 PM" src="https://user-images.githubusercontent.com/1289701/149238803-30f09fd3-7af3-409f-9b82-6107f82a3a49.png">

Reply:
Ah dang... when I don't have `overflow: hidden`, we end up with a massive scroll issue due to how `Skip to main content` is positioned by default.

@ashleemboyer Would an acceptable resolution path here to be position the `Skip to main content` at `top: 4; left: 4` (when focused) in order to avoid this cutoff?

Reply:
Absolutely - here's a screen recording showing the horizontal scroll issue when `overflow: hidden` is removed:

https://user-images.githubusercontent.com/4624598/151180142-d2f2c5e8-338c-4a8a-af2b-2f75a6a95677.mp4

Screen recording is in Safari, but I'm seeing the issue in Firefox and Chrome as well. The `top` & `left` set to `0` is minor compared to the overflow issue - was done as a fix for the cutoff Bryce mentioned above _after_ adding `overflow: hidden` to s

<img width="570" alt="overflow-hidden-issue" src="https://user-images.githubusercontent.com/4624598/151180438-e63deee1-cef6-4a0d-90f7-60c7411f1826.png">
olve for the horizontal scrolling issue:

Reply:
Ah! I see now! So there are a couple of things we can do here instead of `overflow: hidden` in this class.
- Add `overflow-x: hidden` to the `.sidebar` class in `src/layouts/docs/partials/sidebar-sidecar-layout/style.module.css` (it used to be in the `.sidebar` class defined in `src/components/sidebar/style.module.css`).
- Change the default value of `right` in the `.skipToMainContent` class to be `10000px` instead of `-10000px`. This places the skip link on the left side of the sidebar content instead of the right.

I like the second option, and I think it's what I meant to when I implemented this originally!

Reply:
@zchsh looks like this still needs to be addressed! 🙏🏻

Reply:
Ah thanks for flag, lost track of this conversation! Just adjusted with the second option 💯 ✅